User.py

```diff
@@ -9,4 +9,9 @@ class user():
     def __init__ (self,_id,username,password):
         self.id = _id
         self.username = username
-        self.password = password
\ No newline at end of file
+        self.password = password
+        
+    def sample_fun(self):
+        print(self.id)
```

Comment:
you need to print with details using format string

Reply:
ok i changed